### DISCUSSION TITLE
v3.33.5

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#2154] [Announcements]
### Bug Fixes

* **autocmds:** don't map `q` if it's already mapped ([80990eb](https://github.com/AstroNvim/AstroNvim/commit/80990eb0de542c320736cbb168df3f171f2e0212))
* **lsp:** fix disabling lsp semantic tokens ([fb43bd6](https://github.com/AstroNvim/AstroNvim/commit/fb43bd6f575c7faa4fa01bc4b9a1845a6f732c94))

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.33.4...v3.33.5

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.33.5'>v3.33.5</a>.</em>

**Comment by Priest94** [comment]
> 
> ### Bug Fixes
> 
> * **autocmds:** don't map `q` if it's already mapped ([80990eb](https://github.com/AstroNvim/AstroNvim/commit/80990eb0de542c320736cbb168df3f171f2e0212))
> * **lsp:** fix disabling lsp semantic tokens ([fb43bd6](https://github.com/AstroNvim/AstroNvim/commit/fb43bd6f575c7faa4fa01bc4b9a1845a6f732c94))
> 
> 
> **Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.33.4...v3.33.5
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.33.5'>v3.33.5</a>.</em>

~~~~

**Reply by Priest94** [reply]
How are you Boss
